src/Component/Color/index.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import styled from "styled-components";
+
+const Color = styled.div`
+  background-color: ${(props) =>
+    props.background_color ? props.background_color : "white"};
+`;
+export default Color;
```

Comment:
```suggestion
const Color = styled.div`
  background-color: ${(props) =>
    props.backgroundColor ? props.backgroundColor : "white"};
`;
export default Color;
```

변수명은 snake_case 대신 camelCase 를 사용할까요? 🧐